opteryx/results/c6a.4xlarge.json

```diff
@@ -1,56 +1,231 @@
 {
-    "system":  "Opteryx",
-    "date": "2025-07-10",
-    "machine": "c6a.4xlarge",
+    "system": "Opteryx",
+    "date": "2025-07-30",
+    "machine": "c6a.4xlarge, 500gb gp2",
```

Comment:
Why was `500gb gp2` added while the same text does not occur in the other .result files (and generally in no .result file in ClickBench for other databases)?

Reply:
Same as above, based on an older version of the results file - updated